### PR TITLE
ROGER-1103: Update docker-registry role to deploy v2 registry and con…

### PR DESCRIPTION
**Pull request by jordmoz**
…fig.

### CODE REVIEW

File: ansible/roles/docker-registry/tasks/main.yml [hunk 1]
```diff
@@ -8,18 +8,26 @@
     - update_cache
 
 - name: Ensure docker registry dir exists
-  file: path={{ docker_registry_dir }} state=directory owner=root
+  file: path={{ docker_registry2_dir }} state=directory owner=root
   become: yes
 
-- name: Backup registry to docker_backup_dir (if defined) using rsync and cron
-  cron: name="backupdockerregistry" state="present" job="rsync -a {{ docker_registry_dir }}/ {{ docker_backup_dir }}/docker-registry" special_time="daily"
```
Comment (ankanm) [inline ansible/roles/docker-registry/tasks/main.yml]: Do you intend to remove the current cron manually?

File: ansible/roles/docker-registry/templates/config.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,8 @@
+version: 0.1
+storage:
+  filesystem:
+    rootdirectory: /var/lib/registry
+http:
+  addr: 0.0.0.0:5000
+  secret: thisismysecrettherearemanylikeitbutthisoneismine
```
Comment (ankanm) [inline ansible/roles/docker-registry/templates/config.yml.j2]: Can't store this.. will need to use an ansible var and make sure it's available.

File: ansible/roles/docker-registry/tasks/main.yml (hunk 1 above)
Comment (jordmoz) [inline ansible/roles/docker-registry/tasks/main.yml]: Already have.

**Comment by ankanm** [comment]
LGTM other than comments above.

**Pull request merged**